### PR TITLE
Reference management models media v9

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Belongs to #3145 

Updated the parameter names in the constructors from contentType to mediaType, see also [PR ](https://github.com/umbraco/Umbraco-CMS/pull/10689)the CMS.

### CODE REVIEW

File: Reference/Management/Models/Media-v9.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
+---
+versionFrom: 9.0.0
+verified-against: rc001
+meta-title: Media Model
```
Comment (busrasengul) [inline Reference/Management/Models/Media-v9.md:4]:
```suggestion
meta.Title: "Media Model"
```

File: Reference/Management/Models/Media-v9.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+meta.Description: The Media class represents a single item in the media tree.
```
Comment (busrasengul) [inline Reference/Management/Models/Media-v9.md:5]:
```suggestion
meta.Description: "The Media class represents a single item in the media tree."
```

**Review by busrasengul — CHANGES_REQUESTED** [review]

File: Reference/Management/Models/Media-v9.md (hunk 2 above)
Comment (erikjanwestendorp) [reply Reference/Management/Models/Media-v9.md]: @busrasengul Updated 😄

**Comment by jonat123** [comment]
@erikjanwestendorp @busrasengul with the release of RC-3 last week, I'll go ahead and merge in this PR :D

**Pull request merged**